test/profiler/test_profiler.py

```diff
@@ -79,6 +79,12 @@
 
 from torch._C._profiler import _ExperimentalConfig, _ExtraFields_PyCall
 
+def is_gfx94x_arch():
+    if TEST_WITH_ROCM:
+        prop = torch.cuda.get_device_properties(0)
+        if "gfx94" in prop.gcnArchName.split(":")[0]:
+            return True
+    return False
```

Comment:
This helper function was already introduced in https://github.com/ROCm/pytorch/pull/1382/files#diff-e72503c9e3e8766e2d1bacf3fad7b88aa166e0e90a7e103e7df99357a35df8d7R72. Can we try to consolidate instead of repeating this function, by moving it to `torch.testing._internal.common_utils`?

Reply:
Remember a previous instance when `is_navi()` check caused an issue when the hip initialization on the main thread broke few distributed tests. Hope we will not hit the same issue.